pkg/runctl/controller_test.go

```diff
@@ -521,14 +334,15 @@ func Test_Controller_syncHandler_mock(t *testing.T) {
 			controller.testing = &controllerTesting{
 				runManagerStub:             runManager,
 				loadPipelineRunsConfigStub: test.pipelineRunsConfigStub,
+				isUpgradeModeStub:          test.isUpgradeModeStub,
 			}
 			// EXERCISE
-			err := controller.syncHandler("ns1/foo")
+			resultErr := controller.syncHandler("ns1/foo")
 			// VERIFY
 			if test.expectedError != nil {
-				assert.Equal(t, test.expectedError, err)
+				assert.Error(t, resultErr, test.expectedError.Error())
```

Comment:
I think second parameter is the expected value, and the third parameter the actual value. Otherwise the logged diff is negated.

Reply:
Error expects the error as second argument.
See https://pkg.go.dev/gotest.tools/v3/assert#Error

Reply:
You are right. I missed the change from `assert.Equal` to `assert.Error()`.